lib/WP_Auth0_Routes.php

```diff
@@ -317,6 +306,32 @@ private function error_return_array( $code ) {
 					'status' => 403,
 					'error'  => __( 'Forbidden', 'wp-auth0' ),
 				);
+				break;
+		}
+	}
+
+	/**
+	 * Check if a token or token JTI is the same as what is stored.
+	 *
+	 * @param string $authorization - Incoming migration token;
+	 *
+	 * @return bool
+	 */
+	private function valid_token( $authorization ) {
```

Comment:
It possible to end up in some kind of RS256 situation? or it can only ever be HS256?

Reply:
No. Original token was always created with HS256 but this new version compares char to char so the algorithm doesn't matter any longer.